cmd/start_server/start_server.go

```diff
@@ -8,9 +8,25 @@ import (
 	"strconv"
 
 	lsp "github.com/CircleCI-Public/circleci-yaml-language-server/pkg/server"
+	"github.com/rollbar/rollbar-go"
+)
+
+var (
+	RollbarToken = ""
 )
 
 func main() {
+	rollbar.SetToken(RollbarToken)
+	rollbar.SetEnvironment("production")
```

Comment:
Shouldn't this depend on the build? I think we should at least have:

* `development`
* `testing`
* `production`

But also, shouldn't we just disable rollbar in development and testing?